CONTRIBUTING.md

```diff
@@ -4,20 +4,20 @@ OpenBazaar is an open source project. We love contributions! This document outli
 
 ## Issues
 
-Issues should be used primarily for bug reports and directly actionable features. It's OK if you open an issue to ask a question or to start a discussion, however don't take offense if the issue is closed after a few days. General discussion of the protocol is probably better fleshed out in [slack](https://openbazaar-slackin-drwasho.herokuapp.com/).
+Issues should be used primarily for bug reports and directly actionable features. It's OK if you open an issue to ask a question or to start a discussion, however, don't take offense if the issue is closed after a few days. General discussion of the protocol is probably better fleshed out in [slack](https://openbazaar-slackin-drwasho.herokuapp.com/).
 
 ## Go Guidelines
 You must run `gofmt` before each commit. Travis will fail if you do not run it. Most IDEs have the ability to set `gofmt` to run on save or at specified times.
```

Comment:
A recent commit now requires `goimports` instead of `gofmt`. Please update this.